### ISSUE TITLE
Multiple device configuartion

### ISSUE THREAD

**Issue by Grandpa-G** [RobTillaart/AD985X#21]
I am using Arduino Mega 2560. I have 2 devices that need to generate different frequencies almost simultaneously. I have pins to spare. I don't want to use AND logic.

I don't quite understand how to wire.

For the begin, is the first parameter (select), where I have D7 connected? I assume the rest of the pins are obvious.

For the Mega 2560, can I use any pins I want?

**Comment by RobTillaart** [comment]
@Grandpa-G 
Sorry for the late response as I saw your question only now.

Good question, never tried with a MEGA2560.
If you use software SPI for both I expect it to work, 

The D7 must be low to select that 9850. As the MEGA has plenty of IO I would start by giving every 9850 a set of wires of their own, and get them to work in a sort of disjunct mode.  Next step would be to update them simultaneously which would imply sharing the FQUD frequency update line.

Hardware SPI is not tested enough, today I encountered (and sort of fixed) a bug in reset() for the ESP32.
For multi device via hardware there are just too much hardcoded addresses in the code.

Let me know if this helps!
